pippy/hf/_SaveModule.py

```diff
@@ -19,33 +19,35 @@ def _atomic_write(file_contents: str, target_file_path: str, mode="w") -> None:
         mode (str, optional): mode to write file with. Defaults to "w". Only "w" and "a" are supported.
     """
     try:
-        with open(temp_file.name, mode) as f:
+        with open(target_file_path, mode) as f:
```

Comment:
Let's keep the temp_file here, because we want to make sure the target_file_path only exists when writing is finished. Otherwise, we could end up with a corrupted/half written file.